api/src/main/java/org/apache/iceberg/expressions/StrictMetricsEvaluator.java

```diff
@@ -60,7 +57,6 @@ public StrictMetricsEvaluator(Schema schema, Expression unbound) {
   }
 
   public StrictMetricsEvaluator(Schema schema, Expression unbound, boolean caseSensitive) {
-    this.schema = schema;
```

Comment:
Why are we removing this?

Reply:
Ah nvm, I see that it was only used for the purpose of resolving if a field was nested or not